<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>957</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Nimbus Hub</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget {
	background-color: #2c313c;
	color: #e0e0e0;
	font-family: &quot;Noto Sans&quot;;
	font-size: 10pt;
}
QListWidget#navbar {
	background-color: #2c313c;
	border: none;
	padding-top: 10px;
}
QListWidget#navbar::item {
	padding: 12px;
	border-radius: 5px;
	background-color: transparent;
}
QListWidget#navbar::item:selected {
	background-color: #5699ef;
	color: #ffffff;
}
QListWidget#navbar::item:hover {
	background-color: #3c4250;
}
QStackedWidget#mainContent {
	background-color: #2c313c;
}
QListWidget {
	background-color: #3c4250;
	border: 1px solid #505561;
	border-radius: 4px;
	padding: 5px;
}
QListWidget::item:selected {
	background-color: #5699ef;
	color: #ffffff;
	border-radius: 4px;
}
QPushButton {
	background-color: #3c4250;
	border: 1px solid #505561;
	padding: 8px 16px;
	border-radius: 4px;
}
QPushButton:hover {
	background-color: #4a505e;
	border-color: #606570;
}
QPushButton:pressed {
	background-color: #353a45;
}
QPushButton#createProjectButton, QPushButton#newProjectButton {
	background-color: #5699ef;
	color: #ffffff;
	font-weight: bold;
}
QPushButton#createProjectButton:hover, QPushButton#newProjectButton:hover {
	background-color: #6aaaff;
}
QPushButton#createProjectButton:pressed, QPushButton#newProjectButton:pressed {
	background-color: #4588dd;
}
QLineEdit {
	background-color: #3c4250;
	border: 1px solid #505561;
	padding: 6px;
	border-radius: 4px;
}
QLineEdit:focus {
	border: 1px solid #5699ef;
}
QLabel#label, QLabel#label_2 {
	font-size: 16pt;
	font-weight: bold;
	padding-bottom: 10px;
	color: #ffffff;
}
QScrollBar:vertical {
    border: none;
    background: #3c4250;
    width: 10px;
    margin: 0px 0px 0px 0px;
}
QScrollBar::handle:vertical {
    background: #505561;
    min-height: 20px;
	border-radius: 5px;
}
QScrollBar::handle:vertical:hover {
    background: #5699ef;
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QListWidget" name="navbar">
      <property name="minimumSize">
       <size>
        <width>160</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>160</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="iconSize">
       <size>
        <width>24</width>
        <height>24</height>
       </size>
      </property>
      <property name="movement">
       <enum>QListView::Static</enum>
      </property>
      <property name="spacing">
       <number>5</number>
      </property>
     </widget>
    </item>
    <item>
     <widget class="Line" name="line">
      <property name="styleSheet">
       <string notr="true">background-color: #505561;</string>
      </property>
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QStackedWidget" name="mainContent">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="homePage">
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <property name="leftMargin">
         <number>20</number>
        </property>
        <property name="topMargin">
         <number>20</number>
        </property>
        <property name="rightMargin">
         <number>20</number>
        </property>
        <property name="bottomMargin">
         <number>20</number>
        </property>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_6">
          <item>
           <widget class="QListWidget" name="projectListWidget">
            <property name="iconSize">
             <size>
              <width>64</width>
              <height>64</height>
             </size>
            </property>
            <property name="spacing">
             <number>10</number>
            </property>
            <property name="wordWrap">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QWidget" name="detailsPanel" native="true">
            <property name="minimumSize">
             <size>
              <width>300</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>300</width>
              <height>16777215</height>
             </size>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout">
             <item>
              <widget class="QLabel" name="projectNameLabel">
               <property name="font">
                <font>
                 <pointsize>14</pointsize>
                 <weight>75</weight>
                 <bold>true</bold>
                </font>
               </property>
               <property name="text">
                <string>Proje Adı</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignCenter</set>
               </property>
               <property name="wordWrap">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item>
              <layout class="QFormLayout" name="formLayout">
               <item row="0" column="0">
                <widget class="QLabel" name="label_5">
                 <property name="text">
                  <string>Motor Sürümü:</string>
                 </property>
                </widget>
               </item>
               <item row="0" column="1">
                <widget class="QLabel" name="engineVersionLabel">
                 <property name="text">
                  <string>N/A</string>
                 </property>
                </widget>
               </item>
               <item row="1" column="0">
                <widget class="QLabel" name="label_6">
                 <property name="text">
                  <string>Revizyon:</string>
                 </property>
                </widget>
               </item>
               <item row="1" column="1">
                <widget class="QLabel" name="revisionLabel">
                 <property name="text">
                  <string>N/A</string>
                 </property>
                </widget>
               </item>
               <item row="2" column="0">
                <widget class="QLabel" name="label_7">
                 <property name="text">
                  <string>Oluşturulma:</string>
                 </property>
                </widget>
               </item>
               <item row="2" column="1">
                <widget class="QLabel" name="creationDateLabel">
                 <property name="text">
                  <string>N/A</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <spacer name="verticalSpacer_3">
               <property name="orientation">
                <enum>Qt::Vertical</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>20</width>
                 <height>40</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <widget class="QPushButton" name="deleteProjectButton">
               <property name="styleSheet">
                <string notr="true">background-color: #c0392b; color: white;</string>
               </property>
               <property name="text">
                <string>Projeyi Sil</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_7">
          <item>
           <spacer name="horizontalSpacer_3">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="openProjectButton">
            <property name="text">
             <string>Projeyi Aç</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="newProjectButton">
            <property name="text">
             <string>Yeni Proje</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="newProjectPage">
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <property name="leftMargin">
         <number>20</number>
        </property>
        <property name="topMargin">
         <number>20</number>
        </property>
        <property name="rightMargin">
         <number>20</number>
        </property>
        <property name="bottomMargin">
         <number>20</number>
        </property>
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Yeni Proje Oluştur</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="QListWidget" name="templateListWidget">
            <property name="maximumSize">
             <size>
              <width>220</width>
              <height>16777215</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_2">
            <item>
             <layout class="QFormLayout" name="formLayout_2">
              <item row="0" column="0">
               <widget class="QLabel" name="label_3">
                <property name="text">
                 <string>Proje Adı</string>
                </property>
               </widget>
              </item>
              <item row="0" column="1">
               <widget class="QLineEdit" name="projectNameInput"/>
              </item>
              <item row="1" column="0">
               <widget class="QLabel" name="label_4">
                <property name="text">
                 <string>Konum</string>
                </property>
               </widget>
              </item>
              <item row="1" column="1">
               <layout class="QHBoxLayout" name="horizontalLayout_3">
                <item>
                 <widget class="QLineEdit" name="projectPathInput"/>
                </item>
                <item>
                 <widget class="QPushButton" name="browseButton">
                  <property name="text">
                   <string>...</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
             </layout>
            </item>
            <item>
             <spacer name="verticalSpacer_2">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>40</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>288</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <item>
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="cancelCreateButton">
            <property name="text">
             <string>İptal</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="createProjectButton">
            <property name="text">
             <string>Oluştur</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="newsPage"/>
      <widget class="QWidget" name="notesPage"/>
      <widget class="QWidget" name="docsPage"/>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>957</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>